m4_sessions/1810-inning/privacy.md

```diff
@@ -1,24 +1,28 @@
 ## Group Members:
-
+Tim, Stephen, Kiel, Lucas
 
 
 ## Name of Session: 
 
-## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
+## Session Description (Why should students come to this session): (EX: Building a perfectly secure application!)
```

Comment:
So the description..."Come to this session if you want to be able to build a perfectly secure application"???